src/helpers.js

```diff
@@ -28,6 +31,35 @@ function parseJson (value) {
   }
 }
 
+/**
+ * Return false or file name given input string, depending on the `file:` prefix. For
+ * example: `getFile('file:abc')` will return `abc` while `getFile(abc)` returns `false`.
+ *
+ * @param {string|any} str the input string
+ * @returns {boolean|string} the returned value
+ */
+function getFile (str) {
```

Comment:
What is the purpose of the prefix? I _might__ understand if it was a file-url like file:// but still don't see the need.
wouldn't a better requirement just be that the file exists ...?

Reply:
the problem is that ims-jwt lib will also be used in an action to generate a token w/o the file feature, hence we need low latency for that use case and we don't want to check the file system when it's not needed.
Having the file prefix makes the intent clear and avoid us to check if a file exists. An alternative could be to check if the private key string is in a valid format and only if not then read the fs but that complicates the logic.
`file:<path>` is one proposition, which I took from the npm approach:
```
  "dependencies": {
    "bar": "file:../foo/bar"
  }
```

I am fine using another prefix or doing the alternative way but I would really like to avoid the extra fs.exists check